node/pkg/driver/node.go

```diff
@@ -444,12 +494,27 @@ func (d *NodeService) NodeUnpublishVolume(ctx context.Context, req *csi.NodeUnpu
 	if len(target) == 0 {
 		return nil, status.Error(codes.InvalidArgument, "Target path not provided")
 	}
+	targetPathWithHostPrefix := GetPodPath(target)
+
+	logger.Debugf("Check if target file exists %s", target)
+	if !d.NodeUtils.IsPathExists(targetPathWithHostPrefix) {
+		logger.Warningf("Idempotent case: target file %s doesn't exist", targetPathWithHostPrefix)
+		return &csi.NodeUnpublishVolumeResponse{}, nil
+	}
 
-	logger.Debugf("NodeUnpublishVolume: unmounting %s", target)
-	err = mount.CleanupMountPoint(target, d.mounter, true)
```

Comment:
Can you explain why you remove this line? CleanupMountPoint will do all the things you did for you (check the existence, unmount, delete the path/file)

Reply:
@igorgonibm Please answer here before resolving, even if the change is OK, in case we need this information in the future

Reply:
Lue, this message terribly spammed node log.

Reply:
you are right, the log is a problem. I created a issue (https://github.com/kubernetes/utils/issues/135) to ask for a solution, once it is fixed, I will open ticket to discuss it again.